src/components/Insights.tsx

```diff
@@ -48,14 +48,14 @@ export default function Insights({ isAuthorized, data, updateEntryDate, selectio
     })
     .catch(err => {
       setError(true)
-      console.error(err)
     })
 
     return () => {
       setloading(false)
       setError(false)
       setEmptyDay(false)
     }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I am curious about this comment, I saw it in my code and cleaned some things up but wasn't really sure what it means?

Reply:
Yeah! Essentially we're telling ESLint to temporarily ignore the dependency rule for useEffect on the next line of code. It's to be used sparingly, but I threw it in wherever I saw the linter upset with the dependency array because I didn't want to add anything to the array in case it caused looping or new errors. Definitely something we can go back and look at when we have time.